### PR TITLE
community/py-markdown: upgrade to 3.1

### PR DESCRIPTION
**Pull request by fabaff**

### CODE REVIEW

File: community/py-markdown/APKBUILD [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=py-markdown
 _pkgname=Markdown
-pkgver=3.0.1
-pkgrel=1
+pkgver=3.1
+pkgrel=0
 pkgdesc="A Python implementation of Markdown"
 url="http://pythonhosted.org/Markdown/"
 arch="noarch"
```
Comment (maxice8) [inline community/py-markdown/APKBUILD]: BSD isn't an SPDX identifier and is ambiguous

Comment (maxice8) [inline community/py-markdown/APKBUILD]: empty variables can be removed from brevity

Comment (maxice8) [inline community/py-markdown/APKBUILD]: makedepends only need py-setuptools and py3-setuptools

Comment (andypost) [reply community/py-markdown/APKBUILD]: I think this needs separate issue cos a lot of packages affected

**Pull request closed without merging**